src/companies/models/employee.py

```diff
@@ -27,6 +27,7 @@ class Employee(TimestampedModel):
     )
 
     class Meta:
+        ordering = ("fire_date",)
```

Comment:
fire_date is optional field, so it's also not so efficient. Use `created` field.